auth/providers/azure/graph/graph.go

```diff
@@ -72,7 +72,8 @@ var (
 )
 
 const (
-	expiryDelta            = 60 * time.Second
+	// Time delta to refresh token before expiry
+	tokenExpiryDelta       = 300 * time.Second
```

Comment:
Can you also call out this change from 1min to 5min in the PR description?

Reply:
Updated